### PR TITLE
Article XIV

### PR DESCRIPTION
**Pull request by ColinMaudry**

### CODE REVIEW

File: docs/gpa.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This document annotates selected parts of the GPA, with details on how to publis
 ```eval_rst
 .. warning::
 
-   References from Articles VII:4, VII:5, IX:8, IX:9, X:7, X:9, X:11, XIII:2, XIV:1, XVI:2 are not yet added.
+   References from Articles XIII:2, XIV:1, XVI:2 are not yet added.
```
Comment (jpmckinney) [inline docs/gpa.md:15]: Did we do IX:8 and IX:9?

Comment (ColinMaudry) [reply docs/gpa.md:15]: Ah no, there aren't in the file. I will add them (if relevant).

Comment (jpmckinney) [reply docs/gpa.md:15]: I created an issue. PR is approved!

Comment (ColinMaudry) [reply docs/gpa.md:15]: Yes, I'll work on #22. I'll try to suggest guidance.

**Comment by ColinMaudry** [comment]
Why don't you merge?

**Pull request merged**